src/pages/components/split-button.mdx

```diff
@@ -39,6 +39,40 @@ components:
 
 ## How to use it
 
+### Default
+
+The typical usage of a Split Button component.
+
+import SplitButtonDefault from '../../examples/button/SplitButtonDefault.tsx';
+import SplitButtonDefaultCode from '!!raw-loader!../../examples/button/SplitButtonDefault.tsx';
+
+<CodeExample code={SplitButtonDefaultCode}>
+  <SplitButtonDefault />
+</CodeExample>
+
+### Danger
+
+Use danger styling for Split Buttons that enable destructive action.
+
+import SplitButtonDanger from '../../examples/button/SplitButtonDanger.tsx';
+import SplitButtonDangerCode from '!!raw-loader!../../examples/button/SplitButtonDanger.tsx';
+
+<CodeExample code={SplitButtonDangerCode}>
+  <SplitButtonDanger />
+</CodeExample>
+
+### Disabled
```

Comment:
As I think a little more about this… it is possible to have one side and not the other disabled. So it is technically possible to have an enabled primary action, but a disabled chevron button, and vice versa. We even see this in product (I hate it). I don't know if we want to call out that it's possible and then also create a "how to use it well" section? Maybe that's a different PR?

Reply:
My opinion on this one would be to not illustrate too much configuration for disabled usage and let the forthcoming "Disabled patterns" doc address these considerations for best practices more broadly.